slave.mk

```diff
@@ -284,6 +300,50 @@ endif
 export kernel_procure_method=$(KERNEL_PROCURE_METHOD)
 export vs_build_prepare_mem=$(VS_PREPARE_MEM)
 
+###############################################################################
+## Canned sequences
```

Comment:
can you explain what is "Canned sequences"? it is unclear to me.

Reply:
Added detailed comment in the macro